modules/SofaOpenglVisual/SofaOpenglVisual_test/ClipPlane_test.cpp

```diff
@@ -37,8 +39,14 @@ int initMessage(){
 
 int messageInited = initMessage();
 
-class TestClipPlane : public Sofa_test<> {
+class TestClipPlane : public BaseTest {
```

Comment:
If the test is using a scene it may be better to inhertif from BaseSimulationTest...